src/js/components/DataSummary/__tests__/DataSummary-test.tsx

```diff
@@ -10,7 +11,7 @@ import { DataSummary } from '..';
 
 // asserts that AnnounceContext aria-live region and visible DataSummary each have this text
 const expectDataSummary = (message: string) =>
-  expect(screen.getAllByText(message)).toHaveLength(2);
+  expect(screen.getByText(message)).toBeInTheDocument();
```

Comment:
The test assertion changed because our announcer element is now correctly hidden including by testing tools. Previously, the older hiding styles could sometimes make the hidden announcer still "visible" to getAllByText, causing the test to look for two instances